### PR TITLE
Update com.apple.loginwindow.plist

### PR DESCRIPTION
**Pull request by walkintom**
Added "TALLogoutSavesState" key

### CODE REVIEW

File: Manifests/ManifestsApple/com.apple.loginwindow.plist [hunk 1]
```diff
@@ -608,6 +609,22 @@ Availability: Available in macOS 10.13 and later.</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<true/>
+			<key>pfm_description</key>
+			<string>Reopens windows that were open at time of logout</string>
+			<key>pfm_description_reference</key>
```
Comment (kevinmcox) [inline Manifests/ManifestsApple/com.apple.loginwindow.plist:617]: No reason to leave in keys with empty strings.

File: Manifests/ManifestsApple/com.apple.loginwindow.plist [hunk 2]
```diff
@@ -608,6 +609,22 @@ Availability: Available in macOS 10.13 and later.</string>
...
+			<string></string>
+			<key>pfm_documentation_source</key>
```
Comment (kevinmcox) [inline Manifests/ManifestsApple/com.apple.loginwindow.plist:619]: No reason to leave in keys with empty strings.

**Review by kevinmcox — CHANGES_REQUESTED** [review]
Does this key work alone? In the past when I configured this option via script I was also changing the `LoginwindowLaunchesRelaunchApps` key (but reports vary on if it is necessary).

File: Manifests/ManifestsApple/com.apple.loginwindow.plist (hunk 2 above)
Comment (walkintom) [reply Manifests/ManifestsApple/com.apple.loginwindow.plist:619]: Must admit, I never even knew the `LoginwindowLaunchesRelaunchApps` key existed so yeah, `TALLogoutSavesState` does work independently. I'll fix the empty keys.

**Review by relgit — APPROVED** [review]

**Review by kevinmcox — APPROVED** [review]

**Pull request merged**